x-pack/plugins/security_solution/server/lib/timeline/saved_object_mappings/migrations/notes.ts

```diff
@@ -5,39 +5,9 @@
  * 2.0.
  */
 
-import {
-  SavedObjectMigrationMap,
-  SavedObjectSanitizedDoc,
-  SavedObjectUnsanitizedDoc,
-} from 'kibana/server';
-import { timelineSavedObjectType } from '..';
-import { TIMELINE_ID_REF_NAME } from '../../constants';
-import { createMigratedDoc, createReference } from './utils';
-
-export interface TimelineId {
-  timelineId?: string | null;
-}
-
-export const migrateNoteTimelineIdToReferences = (
```

Comment:
This function was renamed and moved to the utils file because it is shared between the notes and the pinned events.